### DISCUSSION TITLE
Introduce Documentation Style Guide rule validation against Package descriptions

### DISCUSSION THREAD

**Discussion by NikRimington** [umbraco/Umbraco.Packages#45] [Ideas] [answered]
It would be really cool if when setting your text for a package page it could be verified against the Umbraco Documentation style guide rules. This shouldn't stop the page being published but instead send the owners a notification email telling them they might want to re-evaluate their content.

I appreciate that posts etc can't be vetted against this, but something like a Package description page should be 😃

**Comment by nul800sebastiaan** [comment]
🙂 it's a lovely idea.. if only our spaghetti could would make it relatively easy! 😉

We're going to focus more on the packages in the near future, I'll put this on the list of ideas to work on for when Our is being worked on, thank! 👍

**Comment by ronaldbarendse** [comment, marked as answer]
We do now provide a way to validate your package on the Marketplace, which includes suggestions if you're missing some details: https://marketplace.umbraco.com/validate.

This doesn't include validating the actual readme/package description on used style guide rules though. You could copy over the Vale styles from the Umbraco docs ([`.value.ini`](https://github.com/umbraco/UmbracoDocs/blob/main/.vale.ini) and [`.github/styles/UmbracoDocs`](https://github.com/umbraco/UmbracoDocs/tree/main/.github/styles/UmbracoDocs) ) and use this tool in your own repository to validate the readme content before publishing the package (see https://docs.umbraco.com/welcome/contribute/style-guide#test-the-docs-yourself).

If you want to continue this discussion, please create a new [Marketplace issue](https://github.com/umbraco/Umbraco.Marketplace.Issues/issues).
